src/content/en/fundamentals/performance/prpl-pattern/index.md

```diff
@@ -221,6 +222,12 @@ The combination of HTTP/2 and HTTP/2 server push provides the _benefits_ of
 bundling (reduced latency) without actual bundling. Keeping resources separate
 means they can be cached efficiently and be shared between pages.
 
+HTTP/2 Push needs to be utilized with care, as it forces data to the browser,
```

Comment:
True. This is very high level in any case, not sure this list needs to be exhaustive here. I am mainly worried about making PUSH look like a golden bullet. But happy to adjust if you think it’s the better choice.